lib/policies/terminate/terminate.js

```diff
@@ -1,3 +1,10 @@
 module.exports = (params) =>
-  (req, res, next) =>
-    res.status(params.statusCode).send(params.message);
+  (req, res, next) => {
+    let payload;
+    if (req.accepts('json')) {
+      payload = { message: params.message };
+    } else {
+      payload = params.message;
+    }
+    res.status(params.statusCode).send(payload);
```

Comment:
I believe, in case is a JSON response, to use `res.status(500).json({ message: params.message });` so that the content type header is set correctly.

Reply:
`send` does the right thing based on the argument.

https://expressjs.com/en/4x/api.html#res.send